lib/std/debug.zig

```diff
@@ -1525,6 +1548,7 @@ pub const ModuleDebugInfo = switch (builtin.os.tag) {
     .uefi, .windows => struct {
         base_address: usize,
         pdb: pdb.Pdb,
```

Comment:
The `pdb`/`dwarf` can be consolidated into a `union (enum) { pdb: pdb.Pdb, dwarf: DW.Dwarfinfo }`.

Reply:
Good idea